### PR TITLE
add  load_model API

### PR DESCRIPTION
**Pull request by FlyingQianMM**
* Add `load_model` function to load a model trained by paddlex
* Height and width in `Resize` API are rename to target_size, which is consistent with RandomResize/BatchRandomResize.
* Self.target_h and self.target_w in `ResizeByShort` API are deleted because they are needless.
* Fix some bugs.
* Add `use_vdl=True` in model.train() for tutorials.
* Reset some parameters like `warmup_step` for detection tutorials.
* Redefine paddleseg `load_pretrained_model()`.

### CODE REVIEW

File: paddlex/cv/models/base.py [hunk 1]
```diff
@@ -56,7 +56,7 @@ def net_initialize(self, pretrain_weights=None, save_dir='.'):
                     os.remove(save_dir)
                 os.makedirs(save_dir)
             if self.model_type == 'classifier':
-                scale = getattr(self.net, 'scale', None)
+                scale = getattr(self, 'scale', None)
```
Comment (will-jl944) [inline paddlex/cv/models/base.py:59]: model应该是没有attribute 'scale'的，在model.net里才有。

Comment (FlyingQianMM) [reply paddlex/cv/models/base.py:59]: model.net里面没有哈，这里跑不通，你看下classifier.py那我重新把params赋给了model

File: paddlex/cv/models/load_model.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+import copy
+import yaml
+import paddlex
+import paddlex.utils.logging as logging
+from paddlex.cv.transforms import build_transforms
+
+
+def load_model(model_dir):
+    if not osp.exists(model_dir):
+        logging.error("model_dir '{}' is not exists!".format(model_dir))
```
Comment (will-jl944) [inline paddlex/cv/models/load_model.py]: does not exist

File: paddlex/cv/models/load_model.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    if not osp.exists(osp.join(model_dir, "model.yml")):
+        raise Exception("There's not model.yml in {}".format(model_dir))
```
Comment (will-jl944) [inline paddlex/cv/models/load_model.py]: There's no model.yml

File: paddlex/cv/transforms/operators.py [hunk 4]
```diff
@@ -325,16 +317,23 @@ def apply(self, sample):
         if self.interp == "RANDOM":
             interp = random.choice(interp_list)
         im_h, im_w = sample['image'].shape[:2]
+        im_short_size = min(im_h, im_w)
+        im_long_size = max(im_h, im_w)
+        scale = float(self.short_size) / im_short_size
```
Comment (will-jl944) [inline paddlex/cv/transforms/operators.py]: 这里是否跟324行统一一下——scale = float(self.short_size) / float(im_short_size)

File: paddlex/cv/transforms/__init__.py [hunk 5]
```diff
@@ -31,3 +32,15 @@ def arrange_transforms(model_type, transforms, mode='train'):
...
     transforms.arrange_outputs = arrange_transform
+
+
+def build_transforms(transforms_info):
+    transforms = list()
+    for op_info in transforms_info:
+        op_name = list(op_info.keys())[0]
+        op_attr = op_info[op_name]
+        if not hasattr(T, op_name):
+            raise Exception("There's no transform named '{}'".format(op_name))
```
Comment (will-jl944) [inline paddlex/cv/transforms/__init__.py:43]: 如果是batch_operator这里也会raise exception

File: paddlex/cv/models/load_model.py (hunk 2 above)
Comment (FlyingQianMM) [reply paddlex/cv/models/load_model.py]: Done. Thanks.

File: paddlex/cv/models/load_model.py (hunk 3 above)
Comment (FlyingQianMM) [reply paddlex/cv/models/load_model.py]: Done. Thanks.

File: paddlex/cv/transforms/operators.py (hunk 4 above)
Comment (FlyingQianMM) [reply paddlex/cv/transforms/operators.py]: Done. Thanks.

File: paddlex/cv/transforms/__init__.py (hunk 5 above)
Comment (FlyingQianMM) [reply paddlex/cv/transforms/__init__.py:43]: model.yml里面不会保存batch_operator，所以build_transform时不会引入batch_operator.

Comment (will-jl944) [reply paddlex/cv/transforms/__init__.py:43]: TODO: 在evaluate函数中检查eval_dataset.transforms中不包含batch transforms

Comment (FlyingQianMM) [reply paddlex/cv/transforms/__init__.py:43]: Done. Thanks

**Pull request merged**